source/fundamentals/connection.txt

```diff
@@ -47,46 +47,45 @@ usage limits, such as max connections, apply to individual
 
 .. _connection-uri:
 
-Connection String
------------------
+Connection URI
+--------------
 
-The **connection string** (also known as a **connection uri**) provides
-instructions on how to connect to a MongoDB deployment. The driver
-URL interprets the connection string to determine where and how
-it should connect to MongoDB as well as how to behave while
-connected. The following example shows each part of the connection URI:
+The **connection URI** provides a set of instructions that the driver uses to
+connect to a MongoDB deployment. It instructs the driver on how it should
+connect to MongoDB and how it should behave while connected. The following
+example explains each part of a sample connection URI:
 
-.. figure:: /includes/figures/connection_string_parts.png
+.. figure:: /includes/figures/connection_uri_parts.png
    :alt: Connection String parts figure
 
-In this example, for the protocol, we use ``mongodb+srv`` which specifies the
-:manual:`DNS Seedlist Connection Format </reference/connection-string/#std-label-connections-dns-seedlist>`.
-This indicates that the hostname following it corresponds to the DNS SRV record of your
-MongoDB instance or deployment. If your instance or deployment does not have a
-DNS SRV record, use ``mongodb`` to specify the :manual:`Standard Connection
-Format </reference/connection-string/#standard-connection-string-format>`.
+In this example, we use the :manual:`Standard Connection String Format
```

Comment:
We don't want to say URI here?

Reply:
I didn't say URI because I didn't want to stray from its actual term, which can confuse readers. But I think it's clear that is the URI format since the intro states thats what we're using and when clicking on the link, it also says its a URI format.